TACC/Core-CMS
# BM-22: BrainMap: Add UTHSC-SA Logo

### Requires

- https://github.com/TACC/Core-CMS-Resources/pull/110

### Overview

Add UT Health San Antonio logo to [BrainMap](https://portal.brainmap.io/about) branding bar.

### Issue

- [BM-22](https://jira.tacc.utexas.edu/browse/BM-22)

### Changes

Update submodule commit (see https://github.com/TACC/Core-CMS-Resources/pull/110).

### Screenshots

~~Pending Design decision.~~[^2] Going with tall logo in position 4 (to balance SGCI logo)

[^2]: Design (M.S., H.P.) has been asked [internally, privately](https://tacc-team.slack.com/archives/GT0JD92JF/p1638392308008400) what position and size to use for the logo.

<details open>
<summary>Dev Suggestion</summary>

<img width="1080" alt="BM-22 Pos 4 Tall - Edit SGCI" src="https://user-images.githubusercontent.com/62723358/144312902-d0886456-e247-46b9-9fc0-b750ea909731.png">

</details>
<details>
<summary>All Options</summary>

__Drop SGCI Logo Text [^1]__

[^1]: When asked about position and size of logo, M.S. (Design) asked [whether we can drop text on SGCI logo (internal)](https://tacc-team.slack.com/archives/GMERCL6P2/p1638390736001300). M.D. answered, "Yes of course you can drop the text". — [2021-12-01 @ 14:43 (internal)](https://tacc-team.slack.com/archives/GMERCL6P2/p1638391391003200)

| Position | Short | Tall |
| - | - | - |
| 4 | <img width="1080" alt="BM-22 Pos 4 Short - Edit SGCI" src="https://user-images.githubusercontent.com/62723358/144313077-20fb3060-1571-43cf-9c18-3973cf7906e2.png"> | <img width="1080" alt="BM-22 Pos 4 Tall - Edit SGCI" src="https://user-images.githubusercontent.com/62723358/144312902-d0886456-e247-46b9-9fc0-b750ea909731.png"> |

__Keep SGCI Logo Text__

| Position | Short | Tall |
| - | - | - |
| 1 | <img width="1080" alt="BM-22 Pos 1 Short" src="https://user-images.githubusercontent.com/62723358/144306713-f6c67e91-2fbb-4ae1-ae92-abc22f612b0d.png"> | <img width="1080" alt="BM-22 Pos 1 Tall" src="https://user-images.githubusercontent.com/62723358/144306720-33a8c935-ecb3-4a19-9727-fce6dc01bdc0.png"> |
| 2 | <img width="1080" alt="BM-22 Pos 2 Tall" src="https://user-images.githubusercontent.com/62723358/144306402-d6ac114b-f69d-4023-9f96-f0a6122633ca.png"> | <img width="1080" alt="BM-22 Pos 2 Short" src="https://user-images.githubusercontent.com/62723358/144306404-42834c84-a774-4826-bc2d-88c585ee4cb9.png"> |
| 3 | <img width="1080" alt="BM-22 Pos 3 Short" src="https://user-images.githubusercontent.com/62723358/144306408-32ddb043-de28-487d-968f-955dab76ad90.png"> | <img width="1080" alt="BM-22 Pos 3 Tall" src="https://user-images.githubusercontent.com/62723358/144306409-012dcff5-2df7-48f7-bb4f-3412f5f36995.png"> |
| 4 | <img width="1080" alt="BM-22 Pos 4 Short" src="https://user-images.githubusercontent.com/62723358/144306420-03bbfb3d-7a13-47d0-9085-a0154b8d6e28.png"> | <img width="1080" alt="BM-22 Pos 4 Tall" src="https://user-images.githubusercontent.com/62723358/144306274-92f7bf67-1700-4dec-bc9a-e6c9ec58e0c8.png"> |

</details>

### Testing

Remote:
- URL: https://pprd.brainmap.tacc.utexas.edu/about/
- Build: https://jenkins01.tacc.utexas.edu/job/Core_CMS/318/
- Deploy: https://jenkins01.tacc.utexas.edu/job/Core_Portal_Deploy/913/

Local:
1. Be familiar with [How to Test CMS Changes](https://github.com/TACC/Core-CMS/wiki/Test-CMS-Changes).
2. Verify image is white on black.
3. Verify image shows in branding bar at `30px` or `35px` (`branding-logo--tall`).
